codegen/cmd.go

```diff
@@ -250,25 +249,27 @@ func (c Command) generateGroup(
 		Header: c.GeneratedHeader,
 	}
 
-	protoTypes, err := render.RenderProtoTypes(grp)
-	if err != nil {
-		return err
-	}
+	for _, grp := range grps {
+		protoTypes, err := render.RenderProtoTypes(*grp)
+		if err != nil {
+			return err
+		}
 
-	if err := fileWriter.WriteFiles(protoTypes); err != nil {
-		return err
-	}
+		if err := fileWriter.WriteFiles(protoTypes); err != nil {
+			return err
+		}
 
-	apiTypes, err := render.RenderApiTypes(grp)
-	if err != nil {
-		return err
-	}
+		apiTypes, err := render.RenderApiTypes(*grp)
+		if err != nil {
+			return err
+		}
 
-	if err := fileWriter.WriteFiles(apiTypes); err != nil {
-		return err
+		if err := fileWriter.WriteFiles(apiTypes); err != nil {
+			return err
+		}
 	}
 
-	manifests, err := render.RenderManifests(c.AppName, c.ManifestRoot, c.ProtoDir, protoOpts, groupOptions, grp)
+	manifests, err := render.RenderManifests(c.AppName, c.ManifestRoot, c.ProtoDir, protoOpts, groupOptions, grps)
```

Comment:
we sohuld just move RenderManifests out rather than changing the whole function signature for this one line

Reply:
Not exactly sure what you mean. RenderManifests needs all groups now, as it combines multiple versions of the same group into one manifest (and multiple versions of the same resource into a single crd).